x-pack/plugins/stack_connectors/public/connector_types/lib/servicenow/translations.ts

```diff
@@ -412,3 +412,32 @@ export const EVENT_ACTION_LABEL = i18n.translate(
     defaultMessage: 'Event action',
   }
 );
+
+export const ADDITIONAL_INFO = i18n.translate(
+  'xpack.stackConnectors.components.serviceNowITOM.additionalInfoLabel',
+  {
+    defaultMessage: 'Additional info',
+  }
+);
+
+export const ADDITIONAL_INFO_HELP = i18n.translate(
+  'xpack.stackConnectors.components.serviceNowITOM.additionalInfoHelpTooltip',
+  {
+    defaultMessage: 'Additional info help',
+  }
+);
+
+export const ADDITIONAL_INFO_HELP_TEXT = i18n.translate(
+  'xpack.stackConnectors.components.serviceNowITOM.additionalInfoHelpTooltipText',
+  {
+    defaultMessage:
+      'The additional info field was pre-filled with relevant internal data. Its value can be extended with additional information.',
```

Comment:
>      'The additional info field was pre-filled with relevant internal data. Its value can be extended with additional 

This makes me wonder if users have to leave the existing fields as-is (and optionally add more) or whether they can change or remove the pre-filled values. If both are allowed, maybe this description is clearer:

The rule automatically generates information about each event. You can change or add more custom fields in JSON format.

Reply:
Thanks Lisa, I'll update it! 😃